jaxrs-tck/pom.xml

```diff
@@ -31,6 +31,8 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
+        <arquillian-bom.version>1.7.0.Alpha9</arquillian-bom.version>
```

Comment:
The is now Beta #10 (https://search.maven.org/artifact/org.jboss.arquillian.junit5/arquillian-junit5-container/1.7.0.Alpha10/jar) since June 21.